### ISSUE TITLE
Static antenna position plot on Session List View

### ISSUE THREAD

**Issue by tchamberlin** [GreenBankObservatory/antenna_plots_v2#5]
Blocked by [Alda sandbox setup](https://github.com/GreenBankObservatory/summer_onboarding_2023/issues/6)

Display one of the plots from #2 on the Alda homepage (`/disk/sessions`). Note: this plot will _not_ be interactive, nor will it be based on the currently-filtered results on the page. It is purely static -- the goal here is mostly to get "a plot" to display "somewhere"

You will need to:

Create an `alda/static/images` folder, into which you place the image file

Modify `alda/disk/templates/disk/session_detail.html` to include something like

```html
<img src="{% static 'images/all_antenna_positions.jpg' %}" alt="Map of all GBT antenna positions, as of summer 2022!">
```

The image should appear between the filter forms and the table (will look goofy, but this keeps things simple)

Work should be done on a branch off of `main`, e.g. `static_ant_plot`

**Comment by tchamberlin** [comment]
Commits are squashed and merged into `kwei_dev`
